demo/popcha-client/web/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+echo "Creating Virtual environment"
+python -m venv venv;
+echo "Entering Virtual environment"
+source venv/bin/activate;
+echo "Installing dependencies"
+pip install -r requirements.txt;
+if [ $# -eq 2 ] ; then
+    echo "Creating config file (This will reset your client id)"
+    printf "{\"client_id\":\"\", \"host_url\": \"$1\", \"host_port\": $2}" > data/config.json
```

Comment:
If it is empty, it will be generated on launch by the app. Since the configuration would not change, the first time `run.sh` should be called with two arguments  (or the `config.json` should be edited).

Reply:
After some discussions, since the server does not implement persistency and it is not scheduled to implement it (It is a test server after all), the current behavior should not be a problem. Since the client won't be able to reuse its client id.